fuzzing/requirements.txt

```diff
@@ -0,0 +1 @@
+absl-py==0.9.0
```

Comment:
Should this be `>=`? This will make it easy for other Bazel repos that need absl-py to co-exist with this one.